model_training/model_builder/model_inference.py

```diff
@@ -10,7 +10,7 @@
     get_dense_features,
     get_pooling_features,
 )
-from data_preparation.tensorrt_utils import TensorRTLayer
+from data_preparation.pytorch_utils import TensorRTLayer
```

Comment:
I think this import is wrong
```suggestion
from data_preparation.pytorch_utils import PytorchLayer
```

Needs changing on line 55 too